package.json

```diff
@@ -15,16 +15,25 @@
     "@emotion/css": "^11.11.2",
     "@emotion/react": "^11.11.3",
     "@emotion/styled": "^11.11.0",
+    "@mdx-js/loader": "^3.0.1",
+    "@mdx-js/react": "^3.0.1",
     "@mui/base": "^5.0.0-beta.30",
     "@mui/icons-material": "^5.15.3",
     "@mui/lab": "^5.0.0-alpha.159",
     "@mui/material": "^5.15.3",
     "@mui/material-nextjs": "^5.15.11",
-    "next": "^14.1.3",
+    "@next/mdx": "^14.2.7",
+    "autoprefixer": "^10.4.20",
+    "next": "^14.2.7",
+    "postcss": "^8.4.44",
     "react": "^18",
-    "react-dom": "^18"
+    "react-dom": "^18",
+    "tailwind-merge": "^2.5.2",
+    "tailwindcss": "^3.4.10",
+    "webpack": "^5.94.0"
```

Comment:
just curious, which packages require us to use webpack directly? I was hoping to keep it behind the scene and managed by Next.js 😆 

source: https://nextjs.org/docs/app/api-reference/next-config-js/webpack

Reply:
@mdx-js/loader is part of the nextjs recommended mdx setup: https://nextjs.org/docs/pages/building-your-application/configuring/mdx

Reply:
Ooh I see! Thanks for the doc 💡 😄 

Hmm maybe we still don't need to have webpack as direct dependency like Vercel's example here?

https://github.com/vercel/next.js/blob/canary/examples/with-mdx/package.json